test/es6/letconst_global_shadow_builtins.baseline

```diff
@@ -1,4 +1,6 @@
 [object Math]
+ReferenceError: Use before declaration

```

Comment:
These cases are actual use before declarations, not redeclarations.  See below the new try/catches replacing the commented out test cases in `letconst_global_shadow_builtins.js`